postprocessing/tests/test_income_mapping.py

```diff
@@ -0,0 +1,64 @@
+import polars as pl
+from pathlib import Path
+import re
+
+import sys
+
+sys.path.append(str(Path(__file__).resolve().parents[1]))
```

Comment:
I had to add this to import the income mapper. Maybe keep it until we have a default env for resstock? @rajeee